itest/lnd_multi-hop_test.go

```diff
@@ -1921,11 +1924,22 @@ func runMultiHopHtlcAggregation(ht *lntest.HarnessTest,
 	// Once bob has force closed, we can restart carol.
 	require.NoError(ht, restartCarol())
 
-	// Mine a block to confirm the closing transaction.
-	ht.MineBlocksAndAssertNumTxes(1, 1)
+	// Mine an empty block first to trigger the anchor cpfp transaction of
```

Comment:
I think we've already tested it in the previous commit so we don't need to test it again?